css-brochure-form/styles.css

```diff
@@ -0,0 +1,67 @@
+div {
+  color: #2b2b2b;
+  font-weight: 600;
+}
+
+h2 {
+  color: #ec5d57;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-weight: 600;
+  letter-spacing: 2px;
+  margin: auto;
+  text-align: center;
+  width: 50%;
+}
+
+.nonbutton {
+  border: solid 1px gray;
+  border-radius: 1em;
+  font-family: 'AtlasGrotesk', sans-serif;
+  font-size: 50%;
+  margin-bottom: 15px;
+  min-width: 500px;
+  min-height: 45px;
+}
+
+input::placeholder {
+  padding-left: 10px;
+  font-weight: 650;
+  color: #999;
+}
+
+label {
+  font-family: 'Brandon Grotesque', sans-serif;
+  padding: 10px;
+  padding-bottom: 5px;
+}
+
+.button {
+  background-color: #ec5d57;
+  border: none;
+  border-radius: 20px;
+  color: #fffcfc;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-weight: 450;
+  padding: 5px 19px;
+}
+
+.container {
+  margin: auto;
+  max-width: 506px;
+}
+
+.submit {
+  text-align: right;
+}
+
+.option-value {
+  min-width: 507px;
```

Comment:
You have a `min-width` set in multiple different places in your code. Rather than setting a width on multiple items to confine them, I recommend setting a width on just the `container` class, and then you can make each form control element in that `div` a `width` of `100%`